util/util.cpp

```diff
@@ -25,29 +29,40 @@ int getSkuNum()
     return 1;
 }
 
-int getSensorDbusTemp(std::string sensorDbusPath)
+double getSensorDbusTemp(std::string sensorDbusPath, bool unitMilli)
 {
     auto bus = sdbusplus::bus::new_default();
     std::string service = getService(sensorDbusPath);
 
+    double value = std::numeric_limits<double>::quiet_NaN();
+
     if (service.empty())
     {
-        return -1;
+        std::cerr << "Sensor input path not mappable to service: " << sensorDbusPath << std::endl;
```

Comment:
Maybe can print this log when DEBUG?

Reply:
Thanks. I can see how this would be annoying if it repeats printing. It still needs to be printed during normal execution, not DEBUG, because it is an important error. I will add a throttling timer to it, similar to the getService() fix I will do.